test/spoof/spoof.go

```diff
@@ -82,72 +85,75 @@ type SpoofingClient struct {
 	RequestInterval time.Duration
 	RequestTimeout  time.Duration
 
-	endpoint string
-	domain   string
-
 	logf logging.FormatLogger
 }
 
-// New returns a SpoofingClient that rewrites requests if the target domain is not `resolveable`.
+// New returns a SpoofingClient that rewrites requests if the target domain is not `resolvable`.
 // It does this by looking up the ingress at construction time, so reusing a client will not
 // follow the ingress if it moves (or if there are multiple ingresses).
 //
 // If that's a problem, see test/request.go#WaitForEndpointState for oneshot spoofing.
-func New(kubeClientset *kubernetes.Clientset, logf logging.FormatLogger, domain string, resolvable bool, endpointOverride string) (*SpoofingClient, error) {
-	sc := SpoofingClient{
-		Client:          &http.Client{Transport: &ochttp.Transport{Propagation: &b3.HTTPFormat{}}}, // Using ochttp Transport required for zipkin-tracing
-		RequestInterval: requestInterval,
-		RequestTimeout:  RequestTimeout,
-		logf:            logf,
+func New(
+	kubeClientset *kubernetes.Clientset,
+	transport *http.Transport,
+	logf logging.FormatLogger,
+	domain string,
+	resolvable bool,
+	endpointOverride string) (*SpoofingClient, error) {
+	// Spoofing
```

Comment:
This comment is redundant or incomplete.

Reply:
It is mirroring the other comment below `Zipkin tracing`, like in the chain of http.RoundTripper, there is first Tracing and then Spoofing. I don't really care, what should be named? Just delete it?

Reply:
Just delete it. Word `Spoofing` in a `spoof.go` is less than usefull.